### DISCUSSION TITLE
Area of a single cell face

### DISCUSSION THREAD

**Discussion by AsimAl-Sofi** [pyvista/pyvista#4270] [Q&A] [answered]
Hi all
I would like to know if there is a way to get the area of a single cell face in Pyvista?
For example, I have the following mesh:
`out = pv.CylinderStructured(radius=(0,3),theta_resolution=10,height=1,z_resolution=2)`
and I would like to know the area of the first face of the first cell, i.e., 
`out.get_cell(0).get_face(0)`
Is that possible?

**Comment by MatthewFlamm** [comment, marked as answer]
One answer is to cast to an unstructured grid first:

```py
out = pv.CylinderStructured(radius=(0,3),theta_resolution=10,height=1,z_resolution=2)
c = out.get_cell(0).get_face(0)
c.cast_to_unstructured_grid().compute_cell_sizes()["Area"]
```

The first face of the first cell in this case is a degenerate quad cell, so it has no area. The second face has a positive area.

I'm not sure if there is a better approach.

**Reply by AsimAl-Sofi** [reply]
that works perfectly
Thanks a lot @MatthewFlamm
